www/js/lib/zimArchive.js

```diff
@@ -172,31 +172,35 @@ define(['zimfile', 'zimDirEntry', 'util', 'utf8'],
     };
     
     /**
-     * Look for DirEntries with title starting with the given prefix (case-sensitive)
+     * Look for dirEntries with title starting with the given prefix (case-sensitive)
      * 
-     * @param {String} prefix
-     * @param {Integer} resultSize
-     * @param {callbackDirEntryList} callback
+     * @param {String} prefix The case-sensitive value against which dirEntry titles (or url) will be compared
+     * @param {Integer} resultSize The maximum number of results to return
+     * @param {Function} callback The function to call with the array of dirEntries with titles that begin with prefix
```

Comment:
```suggestion
     * @param {callbackDirEntryList} callback The function to call with the array of dirEntries with titles that begin with prefix
```
Keeping the {callbackDirEntryList} type (defined in this file) allows some IDEs to know which parameters are supposed to be passed to this callback. It improves auto-completion and code controls of these IDEs